x-pack/plugins/security_solution/public/common/components/discover_in_timeline/use_discover_in_timeline_context.tsx

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useContext } from 'react';
+import { DiscoverInTimelineContext } from './context';
+
+export const useDiscoverInTimelineContext = () => {
+  const discoverContext = useContext(DiscoverInTimelineContext);
+  if (!discoverContext) {
+    const errMessage = `useDiscoverInTimelineContext should only used within a tree with parent as DiscoverInTimelineContextProvider`;
+    /* eslint-disable-next-line no-console */
+    console.error(errMessage);
```

Comment:
The `errMessage` will already be printed in the console when we `throw` it, do we really need to print it twice?

Reply:
You are right.. we do not need it..I will removed it.